src/utils-lodash.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals module */
+
+'use strict';
+
+// This module returns the list of Lo-Dash methods that will be exported to the main "utils" module. It is coded in a
+// way that it can be used by the CKEditor core code in "utils" as well as from Node.js with the configurations for
+// `grunt lodash`.
+//
+// https://lodash.com/docs
+
+( function() {
+	// The list of Lo-Dash methods to include in "utils".
+	// It is mandatory to execute `grunt lodash` after changes to this list.
+	var lodashInclude = [
+		/**
+		 * Assigns own enumerable properties of source object(s) to the destination object.
+		 *
+		 * From Lo-Dash: https://lodash.com/docs#assign (alias)
+		 *
+		 * @member utils
+		 * @method extend
+		 *
+		 * @param {Object} object The destination object.
```

Comment:
It would be certainly easier to maintain... so, how do we want this? Like the following?

```
		/**
		 * See Lo-Dash: https://lodash.com/docs#assign (alias)
		 *
		 * @member utils
		 * @method extend
		 */
		'extend'
```